reo/models.py

```diff
@@ -191,7 +207,7 @@ class LoadProfileModel(models.Model):
     # Inputs
     run_uuid = models.UUIDField(unique=True)
     doe_reference_name = ArrayField(models.TextField(null=True, blank=True), default=list)
-    annual_kwh = ArrayField(models.FloatField(null=True, blank=True), default=list)
```

Comment:
@t-kwasnik is this change correct and will it work with the migrations?

Reply:
Yes, we talked about converting annual kwh back to a float for simplicity

we have a custom migration here to handle the data: https://github.com/NREL/REopt_Lite_API/blob/chpbeta/reo/migrations/0086_annual_loads_to_float.py

and another to "change" the field type for django to pick up on:
https://github.com/NREL/REopt_Lite_API/blob/chpbeta/reo/migrations/0087_auto_20201218_2121.py